src/pages/home/home.jsx

```diff
@@ -1,3 +1,4 @@
+import { NavLink } from 'react-router-dom';
 import { CategoryCard } from '../../components/categoryCard/categoryCard';
 import { Navbar } from '../../components/navbar/navbar';
 import { Sidebar } from '../../components/sidebar/sidebar';
```

Comment:
You can create an index.js file, and import all the components through index.js.

Reply:
ok thank you